p/scripts/main.js

```diff
@@ -258,7 +253,7 @@ function toggleContent(new_active, old_active) {
 
 	if (context['sticky_post']) {
 		var prev_article = new_active.prevAll('.flux'),
-		    new_pos = new_active.position().top,
```

Comment:
I use spaces instead of tabs to align variables but it's opened to discussion. We may add this to coding style rules.